js_modules/dagit/packages/core/src/sensors/EditCursorDialog.tsx

```diff
@@ -0,0 +1,81 @@
+import {gql, useMutation} from '@apollo/client';
+import {TextArea} from '@blueprintjs/core';
+import {ButtonWIP, DialogBody, DialogFooter, DialogWIP} from '@dagster-io/ui';
+import * as React from 'react';
+
+import 'chartjs-adapter-date-fns';
+
+import {SharedToaster} from '../app/DomUtils';
+import {SensorSelector} from '../types/globalTypes';
+
+export const EditCursorDialog: React.FC<{
+  cursor: string;
+  sensorSelector: SensorSelector;
+  onClose: () => void;
+}> = ({sensorSelector, cursor, onClose}) => {
+  const [cursorValue, setCursorValue] = React.useState(cursor);
+  const [isSaving, setIsSaving] = React.useState(false);
+  const [requestSet] = useMutation(SET_CURSOR_MUTATION);
+
+  const onSave = async () => {
+    setIsSaving(true);
+    const {data} = await requestSet({
+      variables: {sensorSelector, cursor: cursorValue},
+    });
+    if (data?.setSensorCursor.__typename === 'Sensor') {
+      SharedToaster.show({message: 'Set cursor value', intent: 'success'});
+    } else {
+      SharedToaster.show({message: 'Could not set cursor value', intent: 'danger'});
+    }
```

Comment:
similar here - are we swallowing, should we display? just thinking about debugging if this fails

I guess the new capture_errors behavior in python would show the error in the dagit logs maybe?